ShoppingCart-topic1/src/MailList.java

```diff
@@ -0,0 +1,35 @@
+import java.util.ArrayList;
+
+public class MailList implements List {

```

Comment:
Take this into account if you will include this feature in the final project, if not is only a comment, you don't need to update this code